### ISSUE TITLE
The "main" field cannot contain minified files

### ISSUE THREAD

**Issue by ain** [interactive-pioneers/iptools-jquery-modal#44]
Currently latest Bower `1.7.9` considers minified main files as invalid meta in Bower package descriptor and throws a warning of 

```
bower invalid-meta  The "main" field cannot contain minified files
```

The question is, how should we approach this? IMO in our RequireJS-driven systems minified files still make sense.

**Comment by Friedrich-Mocker** [comment]
Option 1: Dont Minify the files
Option 2: Trick bower by not naming it *.min.js
Option 3: ?

**Comment by ain** [comment]
Option 3 would be to bannerise (add that © header) to unminified file and include it in distribution.

I'd actually support the simplest idea of changing `main` path to `src/`. Most applications concat and minify on their own already and if not, people can use RequireJS etc. and point it to load minified file in distribution instead.

**Comment by Friedrich-Mocker** [comment]
What if im using less as a precompiler, then i cant use the uncompiled scss.

**Comment by ain** [comment]
SCSS is really unrelated, there's css compiled into distribution.

**Comment by GeorgMeyer23** [comment]
jup, let's skip legacy support with compiled files...

**Comment by ain** [comment]
Resolved in a55cf70.
